registry/api/v2/errors.go

```diff
@@ -15,6 +15,7 @@ var (
 		Message: "The operation is unsupported.",
 		Description: `The operation was unsupported due to a missing
 		implementation or invalid set of parameters.`,
+		HTTPStatusCode: http.StatusMethodNotAllowed,
```

Comment:
Exactly.  As an aside, one thing I've seen done is an endpoint to describe the available API.  https://api.github.com/ being an example.